### ISSUE TITLE
Add more unit tests for Storage and Network

### ISSUE THREAD

**Issue by TRomesh** [cloudlibz/nodecloud-cli#24]
- Follow the same pattern used in the existing unit tests

**Comment by duc0905** [comment]
Hi, I just want to be clear about the task.
The goal of the task is creating unit tests for others providers, gcp and azure, for storage and network services. So basically, I have to add more test in storage.test.js and network.test.js and create Gcpmock.js and Azuremock.js file for those tests.
Please tell me whether I am mistaken.

**Comment by TRomesh** [comment]
> Hi, I just want to be clear about the task.
> The goal of the task is creating unit tests for others providers, gcp and azure, for storage and network services. So basically, I have to add more test in storage.test.js and network.test.js and create Gcpmock.js and Azuremock.js file for those tests.
> Please tell me whether I am mistaken.

@duc0905 Yes you can create mock files and write tests.  You can write more test for [network.test.js](https://github.com/cloudlibz/nodecloud-cli/blob/master/test/network.test.js)
by looking for the methods which has no test written  under [network.js](https://github.com/cloudlibz/nodecloud-cli/blob/master/lib/network.js). 
i.e  
 [network.test.js](https://github.com/cloudlibz/nodecloud-cli/blob/master/test/network.test.js) has only 4 tests but [network.js](https://github.com/cloudlibz/nodecloud-cli/blob/master/lib/network.js) has more methods, Write test for the reaming methods.
